dc-cudami-admin/src/main/resources/messages.properties

```diff
@@ -58,6 +58,7 @@ creation_info=Creation information
 creation_info_creator=Creator
 date=Date
 date.format=MM/dd/yyyy
+date.format.bc=MM/dd/yyyy 'v.Chr'
```

Comment:
das sollte englisch sein (B.C.?) und in deutsch fehlt es, s. unten

Reply:
ist jetzt gefixed